noxfile.py

```diff
@@ -81,7 +81,7 @@ def lint(session):
     session.run("pylint", "fawltydeps")
     session.run(
         "pylint",
-        "--disable=missing-function-docstring,invalid-name,redefined-outer-name",
+        "--disable=missing-function-docstring,invalid-name,redefined-outer-name,too-many-lines",
```

Comment:
What's causing this?

Reply:
`test_cmdline.py` has grown past 1000 lines 😱

Related to what I mentioned in today's meeting. Also excited to see what @mknorps comes up with in her test PR.